lib/challenge.js

```diff
@@ -161,9 +190,24 @@ function prompt8() {
  */
 
 const starWars = ["Han", "C3PO", "R2D2", "Luke", "Leia", "Anakin"];
-
+//first print each index name-- as it counts up the array, it should tell us what that is.
 function prompt9() {
-  // YOUR CODE HERE
+  //lets do a for loop
+// for(i = 0; i < 6; i++){
+//   console.log(starWars[i]);
+// }
+//lets do a while loop
+// let i = 0
+// while (i < 6) {
+//   console.log(starWars[i]);
+//   i++
+// }
+//lets now print their name with the index
+// let i = 0
+// while(i < 6){
+//   console.log(`${i}. ${starWars[i]}`);
+//   i++
+// }
 }
```

Comment:
I like the variations!  Are you able to write these dynamically?  In other words instead of hard coding the length to 6 is there a way to write this to accept arrays of varying sizes without having to change the 6 to match the array by hand? (resubmission not required)